backend/app/services/implementations/log_records_service.py

```diff
@@ -246,6 +270,7 @@ def delete_log_record(self, log_id):
             raise Exception(
                 "Log record with id {log_id} not found".format(log_id=log_id)
             )
+        log_record_to_delete.residents = []
```

Comment:
why is log_record_to_delete.residents getting set to an empty list before getting deleted? Also, do the rows on the LogRecordResidents also need to get deleted?

Reply:
when you set this to an empty list an call `commit()`, the ORM automatically deletes all associated rows in `LogRecordResidents`